README.md

```diff
@@ -78,9 +74,14 @@ To install ipython/jupyter dependencies for the examples:
 
     pip install -r data_api/requirements-jupyter.txt
 
+You can view examples using a local ipython/jupyter notebook:
```

Comment:
Many of these are broken against the current API. Rather than try to debug them all for this PR, I suggest taking this out and adding a JIRA issue to fix them.